Main.java

```diff
@@ -1,39 +1,22 @@
-import java.util.Collections;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.Scanner;
 
 public class Main {
-    public static void main(String[] args) {
-
-        //TODO switch from using static methods to public methods from Deck class.
-        LinkedList<Card> deck = new LinkedList<>();
-
-        initialiseDeck(deck);
 
-        System.out.println(getRandomCard(deck));
+    public static int sum = 0;
 
-    }
+    public static void main(String[] args) {
+        Deck myCard = new Deck();
+        myCard.initialise();
 
-    private static Card getRandomCard(LinkedList<Card> deck) {
-        Collections.shuffle(deck);
-        return deck.pop();
-    }
+        System.out.println(myCard.hit());
```

Comment:
Probably it would be better to get card to local variable. Or even better to List<Card> hand to have the information about every card that was drew.